src/run_releasenotes_check.py

```diff
@@ -23,18 +25,33 @@
 def main() -> int:
     args = ReleaseNotesCheckArgs()
     console.configure(level=args.logging_level)
-    manifest_file = InputManifest.from_file(args.manifest)
-    BUILD_VERSION = manifest_file.build.version
-
+    manifests: List[InputManifest] = []
     # storing temporary release notes for testing purposes
     BASE_FILE_PATH = "release_notes_workflow/results"
+
+    for input_manifests in args.manifest:
+        manifests.append(InputManifest.from_file(input_manifests))
+
+    #  Assuming that the OS and OSD manifests will be provided for the same release version
+    BUILD_VERSION = manifests[0].build.version
+
     table_filename = f"{BASE_FILE_PATH}/release_notes_table-{BUILD_VERSION}.md"
     urls_filename = f"{BASE_FILE_PATH}/release_notes_urls-{BUILD_VERSION}.txt"
 
+    def check_if_exists_then_delete(file_path: List[str]) -> None:
+        for file in file_path:
+            if os.path.exists(os.path.join(os.path.dirname(__file__), file)):
+                print(f"file {file} exists. Deleting")
+                os.remove(os.path.join(os.path.dirname(__file__), file))
+            else:
+                logging.info(f"The file {file} does not exist, creating new.")
```

Comment:
Shall we have a param to allow user to override or keep the old generated copy?

Reply:
Seems redundant. If a user wants to generate a new one for the same version, better to overwrite the existing one, in case user wants to generate for another version, a new file will be created. There is already an `--output` option to save files elsewhere, the user can use that one to keep older files.

Reply:
Got it. Make sense. Close now.